commands/operator-sdk/cmd/up/local.go

```diff
@@ -77,10 +79,18 @@ const (
 )
 
 func upLocalFunc(cmd *cobra.Command, args []string) {
-	mustKubeConfig()
-
 	log.Info("Running the operator locally.")
 
+	// get default namespace to watch if unset
+	if namespace == "#invalid#" {
+		_, defaultNamespace, err := k8sInternal.GetKubeconfigAndNamespace(kubeConfig)
```

Comment:
What if namespace is set should we still setup the getting of kubeconfig part? I could be missing something as well!

Reply:
No, here we only use that function just to get the default namespace specified by the kubeconfig and we discard the kubeconfig itself using `_`. The operators get the kubeconfigs themselves, so we don't get the kubeconfig.

Reply:
Ah that makes sense, thanks for clarifying!